components/now-client.tsx

```diff
@@ -26,11 +29,7 @@ const mutation = gql`
 `;
 
 const UpdateTimeButton = () => {
-  const client = getClient();
-
-  const [updateTime, { loading, error }] = useMutation(mutation, {
-    client,
-  });
+  const [updateTime, { loading, error }] = useMutation(mutation);
```

Comment:
As `getClient` should return a per-request shared instance in most situations, it should be fine to just have a wrapper around the original `useMutation/useClient` versions and call `getClient` transparently inside of that.